Dockerfile

```diff
@@ -12,14 +12,14 @@ COPY go.* .
 # Copy the required source directories
 COPY api api
 COPY controllers controllers
-COPY ibu-imager ibu-imager
+COPY lca-cli lca-cli
 COPY internal internal
 COPY main main
 COPY utils utils
 
 # Build the binaries
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -mod=vendor -a -o build/manager main/main.go
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -mod=vendor -a -o build/ibu-imager main/ibu-imager/main.go
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -mod=vendor -a -o build/lca-cli main/lca-cli/main.go
```

Comment:
This will need to be coordinated with the downstream build. cc @rauhersu

Reply:
yeah, ping me when you decide to merge this and I will align midstream to these  changes.